log/README.md

```diff
@@ -57,11 +57,13 @@ The checkpoint body is of the form:
 The first 3 lines of the body **MUST** be present in all Checkpoints.
 
 * `<Origin string>` should be a unique identifier for the log identity which issued the checkpoint.
-  The exact format of this is left undefined, but best practice is to use a URI-like strucutre like
-  `<dns_name>[/<suffix>]`, where the log operator controls `<dns_name>`, e.g `example.com/log42`.
-This structure reduces the likelihood of origin collision,
-  and gives clues to humans about the log operator and what is in the log. The suffix is optional
-  and can be anything. It is used to disambiguate logs owned under the same prefix.
+  The format SHOULD be a URI-like strucutre like `<dns_name>[/<suffix>]`, where the log operator
```

Comment:
Were you working on an old copy? The "strucuture" typo and "With a globally unique..." line below came back :)

Reply:
Oops, thanks!